packages/webapp/next.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = withTM(
       dest: 'public',
       disable: process.env.NODE_ENV === 'development',
       runtimeCaching,
+      buildExcludes: [/react-syntax-highlighter|reactSyntaxHighlighter/]
```

Comment:
To further optimize I would have to implement [custom register logic](https://github.com/shadowwalker/next-pwa?tab=readme-ov-file#:~:text=env.NODE_ENV%20%3D%3D%3D%20%27development%27-,register,-%3A%20boolean%20%2D%20whether%20to), which would allow to only register service worker when actually inside ios/android pwa. That would eliminate all the precache requests and save a bunch of bandwidth.